src/gas/fetchBlockFeeHistory.test.ts

```diff
@@ -333,6 +333,12 @@ describe('fetchBlockFeeHistory', () => {
     const latestBlockNumber = 3;
     const numberOfRequestedBlocks = 3;
 
+    beforeEach(() => {
```

Comment:
This should have been there all along and for some reason it was not causing the test to fail.